src/com/facebook/buck/ide/intellij/ModuleBuildContext.java

```diff
@@ -44,6 +44,7 @@
 
   private Optional<IjModuleAndroidFacet.Builder> androidFacetBuilder;
   private ImmutableSet.Builder<Path> extraClassPathDependenciesBuilder;
+  private ImmutableSet.Builder<Path> extraModuleDependenciesBuilder;
```

Comment:
I was trying to keep it generalized so that other build steps could leverage this synthetic modules functionality. If you prefer, I can just use a single Path.